apps/pythinker-web/src/style.css

```
@@ -567,6 +567,8 @@ html[data-color-scheme=dark][data-accent=mono] {
     --p-focus-ring: 0 0 0 3px var(--color-accent-soft);
     --p-focus-ring-strong: 0 0 0 3px var(--color-accent-soft), 0 0 0 1px var(--color-accent);
     --p-selection: rgba(23, 131, 255, .2);
+    --color-code-selection: rgba(23, 131, 255, .346);
+    --color-code-selection-text: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)